programs/lib/include/flexpret_assert.h

```diff
@@ -4,8 +4,10 @@
 #include "flexpret_io.h"
 #include <stdbool.h>
 
+#undef assert
+
 #define assert(cond) do {   \
```

Comment:
Good idea, I'll look into that!

Reply:
In the most recent commits I made _fp_abort and _fp_finish into macros that use __FILE__ and __LINE__ to print where they are run from. Also, since assert uses _fp_abort it also does that. Furthermore, I added a second argument to assert which is the reason it fails. When the assert fails it will print that string in red. So when writing the line of code:

`assert(0, "Got zero");`

It will result in

`[0]: <file>.c: <line>: Abort: Got zero\n`

In the emulator terminal. Where 'Abort: Got zero' is in red.

However, an issue to aware about is that it looks like including some standard library stuff after our stuff might override out macros. Specifically, I noticed that

`#include <flexpret.h>
#include stdlib.h`

Gave an incorrect definition of the assert macro.